fastdds_monitor.pro

```diff
@@ -88,3 +88,11 @@ else:unix: LIBS += -L$$PWD/../../install/fastdds-statistics-backend/lib/ -lfastd
 
 INCLUDEPATH += $$PWD/../../install/fastdds-statistics-backend/include
 DEPENDPATH += $$PWD/../../install/fastdds-statistics-backend/include
+
```

Comment:
Check in line above <81,82> if `/include` is needed

Reply:
Addressed in 1fe2bed